expts/neurips2023_configs/config_small_gated_gcn.yaml

```diff
@@ -295,10 +295,12 @@ metrics:
   tox21:
     - name: auroc_flat
       metric: auroc_ipu
+      task: binary
```

Comment:
The `multitask_handlig` should change to `mean-per-label` if we want to match the litterature

Reply:
Do we wanna change all the auroc_flat to `mean-per-label` or just for the tox21 case?